src/crawlee/project_template/templates/main.py

```diff
@@ -20,7 +20,11 @@
 # % endif
 # % endblock
 # % endfilter
+{% if self.pre_main is defined %}
```

Comment:
Interesting trick! Could you use line expressions (`# % if ...`) here like in the rest of the file?

Reply:
Done